subsystems/PistonControl.java

```diff
@@ -0,0 +1,28 @@
+package org.frc5587.lib.subsystems;
+
+import edu.wpi.first.wpilibj.DoubleSolenoid;
+import static edu.wpi.first.wpilibj.DoubleSolenoid.Value.*;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+public abstract class PistonControl extends SubsystemBase{
+    private DoubleSolenoid[] doubleSolenoidPistons;
+
+    public PistonControl(int[] pistonIDS) {
+        for(int i = 0; i < pistonIDS.length; i+=2) {  // for the amount of numbers in the pistonIDS array defined in constants
+            doubleSolenoidPistons[i] = new DoubleSolenoid(i, i+1);  // create a new double solenoid
```

Comment:
I think `pistonIDs` should be a 2D array, so then each row would represent both ports of the double solenoid. So the array would be 2 by the number of solenoids.